Mage/src/main/java/mage/abilities/dynamicvalue/common/CountersSourcePlayerCount.java

```diff
@@ -0,0 +1,65 @@
+package mage.abilities.dynamicvalue.common;
+
+import mage.abilities.Ability;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.effects.Effect;
+import mage.counters.Counter;
+import mage.counters.CounterType;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+
+import java.util.UUID;
+
+public class CountersSourcePlayerCount implements DynamicValue {
+
+    private final CounterType counterType;
+
+    /**
+     * Number of counters of any type on the source permanent
+     */
+    public CountersSourcePlayerCount() {
+        this((CounterType) null);
```

Comment:
Pointless to add a constructor for null if no logic is implemented to support that use case.

I don't believe there are any cards that care about total number of counters of any type on a player. So safe to remove that part of the logic.